Domains/Jobs/Traits/Worker.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @file Domains/Jobs/Traits/Worker.php
```

Comment:
This class mirrors the defaults in the constructor to [`WorkerOptions`](https://laravel.com/api/5.8/Illuminate/Queue/WorkerOptions.html#method___construct), right? Are we overriding these anywhere? If not, and this is for future potential use but not a current use case yet, I'd favour not adding it.

Reply:
This trait enables our Job Model (Domains/Jobs/Job.php) to customized the parameters for the WorkerOptions inside the PKPQueueProvider (classes/core/PKPQueueProvider.inc.php).

Reply:
OK, but are there cases yet where this is done?

Reply:
(^ @henriqueramos)